helm/prometheus-meta-operator/templates/prometheus-rules/deployment.workload-cluster.rules.yml

```diff
@@ -22,6 +22,17 @@ spec:
         severity: page
         team: batman
         topic: releng
+    - alert: WorkloadClusterDeploymentNotSatisfiedLudacris
```

Comment:
Can we maybe somehow make this an SLO thingy?

Reply:
Maybe? It is a single pod in the deployment, this would fire if it is down for 30min. I guess the upside is SLO would also catch a flappy metrics server?

Reply:
I think using this PR as a reference should be pretty straight forward:
https://github.com/giantswarm/prometheus-meta-operator/pull/588/files

LMK if we can pair on it or something

Reply:
So after looking into it a bit more, I think the better approach would be to create some class of our components which we want to be SLO alerts (like [managed apps](https://github.com/giantswarm/prometheus-meta-operator/blob/master/helm/prometheus-meta-operator/templates/recording-rules/service-levels.rules.yml)). There doesn't seem to be a clean way to do it for a single component at this point (because IIUC alert routing is still not done by the usual team label but based on the component when the alert reaches opsgenie).
So I'd suggest we go with this for now so we've at least got it in place and then move our alerts to SLO as part of a bigger conversation about which and how.